contracts/base/ModuleManager.sol

```diff
@@ -463,6 +458,12 @@ abstract contract ModuleManager is Storage, EIP712, IModuleManagerEventsAndError
         return _getAccountStorage().validators.contains(validator);
     }
 
+    /// @dev Checks if there is at least one validator installed.
+    /// @return True if there is at least one validator, otherwise false.
+    function _hasValidators() internal view returns (bool) {
+        return _getAccountStorage().validators.getNext(address(0x01)) != address(0x01);
```

Comment:
is this the optimised version of below?

require(!(prev == address(0x01) && validators.getNext(validator) == address(0x01)), CannotRemoveLastValidator());

Reply:
that's an internal func, to be able to check within `initializeAccount` without having to import sentinellib on nexus.sol

Reply:
no i mean this is the replaced code for above.

Reply:
yes the one suggested by cantina

Reply:
on nexus it is fine.
require(_hasValidators(), MissingValidator());

but on ModuleManager one, it should still be CanNotRemoveLastValidator

Reply:
I'll revert it on the ModuleManager, but I would've preferred to have a unique Error used twice.